src/components/App.js

```diff
@@ -1,28 +1,38 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-// eslint-disable-next-line no-unused-vars
 import Calculate from '../logic/Calculate';
 
 class App extends React.Component {
   constructor() {
     super();
 
     this.state = {
-      result: '0',
+      total: null,
+      next: null,
+      operation: null,
+    };
+
+    this.handleClick = clickedButton => {
+      this.setState(prevState => Calculate(prevState, clickedButton));
+    };
+
+    this.renderDisplay = values => {
+      const { total, next, operation } = values;
+      if (!total && !next && !operation) return '0';
+      return `${total || ''} ${operation || ''} ${next || ''}`;
```

Comment:
Interesting 😊.

What's the reason for having those assigned on the constructor instead of defined as class methods? 

This might be more clear as it is explicit that those are class methods and you are not overloading your constructor, but not really sure about your thoughts about this 🤔 or why you chose to do it the way you have it. Maybe I'm missing something:

```js
  constructor() {
    super();

    this.state = {
      total: null,
      next: null,
      operation: null,
    };

  }

  handleClick = clickedButton => {
    this.setState(prevState => Calculate(prevState, clickedButton));
  };

  renderDisplay = values => {
    const { total, next, operation } = values;
    if (!total && !next && !operation) return '0';
    return `${total || ''} ${operation || ''} ${next || ''}`;
  };
```